### DISCUSSION TITLE
Changing the background color of `md-list-item`?

### DISCUSSION THREAD

**Discussion by knttnk** [material-components/material-web#4332] [Q&A] [answered]
Hi.

Is there any way to change the background color of `md-list-item`?

I'm trying to make it transparent, but this code isn't working.
```html
<md-list-item style="background: transparent;">
</md-list-item>
```

**Comment by asyncliz** [comment, marked as answer]
Should be able to set `--md-list-item-list-item-container-color`

CC @e111077 I think we should rename those and remove the extra "list-item" since we're separating the tokens into separate "list" and "list-item" token sets.

**Comment by knttnk** [comment]
It works. Thank you!
